app/helpers/ect_helper.rb

```diff
@@ -29,6 +29,16 @@ def ect_status(_ect)
     govuk_tag(text: 'In progress', colour: 'green')
   end
 
+  # @param ect [ECTAtSchoolPeriod]
+  def ect_programme_type(ect)
+    case ect.programme_type
+    when 'provider_led' then 'Provider-led'
+    when 'school_led' then 'School-led'
+    else
+      'Unknown'
+    end
+  end
```

Comment:
I guess we will be doing this in several other places around the codebase.
Would it be worth defining a `class ProgrammeType` with a constant with all the valid programme types, methods like `#to_s`, etc? so it can be reused anywhere in the code?

Reply:
I wouldn't be surprised. I can look at that.